neo/public/javascripts/timeseries/controllers.js

```diff
@@ -34,7 +34,7 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
     countDiv.id = "count-div";
     countDiv.title = "Display the count information of Tweets";
     countDiv.innerHTML = [
-      "<p id='count'>{{ currentTweetCount | number:0 }}<span id='count-text'>&nbsp;&nbsp;of&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p>",
+      "<p id='count'>{{ currentTweetCount | number:0 }}<span id='count-text'>&nbsp;&nbsp;of&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p>",

```

Comment:
It will look like this. I used these spaces to fill the gap. E.g. "of      " -> "tweets"
![screen shot 2017-03-04 at 13 26 43](https://cloud.githubusercontent.com/assets/12385178/23582400/3d782b8e-00de-11e7-9866-807cdaed1a96.jpg)